backend/upload/management/commands/upload-lassy.py

```diff
@@ -20,6 +20,10 @@ def userinputyesno(prompt, default=False):
     else:
         default_view = ' [y/N]'
     sys.stderr.write(prompt + default_view + '\n')
+    global use_defaults
```

Comment:
I think this would be better handled by setting an instance attribute on the `Command` object and checking it in the two places where you call `userinputyesno()`